plutus-core/untyped-plutus-core/src/UntypedPlutusCore/Core/Type.hs

```diff
@@ -77,9 +80,15 @@ data Term name uni fun ann
     deriving anyclass (NFData)
 
 -- | A 'Program' is simply a 'Term' coupled with a 'Version' of the core language.
-data Program name uni fun ann = Program ann (TPLC.Version ann) (Term name uni fun ann)
+-- | A 'Program' is simply a 'Term' coupled with a 'Version' of the core language.
+data Program name uni fun ann = Program
```

Comment:
If you're going to do this, we should do it for the `Program` types for the other ASTs too.

Reply:
i think only plutusir ast is missing? i will add it